### DISCUSSION TITLE
Need help, how to upgrade Netbox from v2.11.2 to the latest release

### DISCUSSION THREAD

**Discussion by uc33260** [netbox-community/netbox#9646] [Q&A]
Hi team,

I'm a beginner with Netbox and I have release 2.11.2 installed.

I need a documentation that explain how to upgrade to the latest release please.

Thank you for help

**Comment by jeremystretch** [comment]
https://docs.netbox.dev/en/stable/installation/upgrading/

**Reply by uc33260** [reply]
Thank you for the link,

I upgraded like mentionned on the document, but I have this error message when I try to connect to Netbox from a browser:

502 Bad Gateway
nginx/1.18.0 (Ubuntu)

Question: "netbox-X.Y.Z" on the upgrading document, does I need to replace X.Y.Z by the new release or the existing one ?
